layers/+lang/haskell/packages.el

```diff
@@ -26,7 +26,7 @@
     hindent
     hlint-refactor
     intero
-    dante
+    (dante :toggle (version<= "25" emacs-version))
```

Comment:
@robbert-vdh  It read as "add `dante` when version 25 is less then or equal to the Emacs version."  Probably  :thinking: